addons/spreadsheet/static/src/global_filters/components/filter_value/filter_value.js

```diff
@@ -115,6 +117,35 @@ export class FilterValue extends Component {
         this.props.setGlobalFilterValue(id, { operator, strings: value });
     }
 
+    onTargetValueNumericInput(id, value) {
+        const operator = this.filterValue?.operator ?? this.getDefaultOperator();
+        const newFilterValue = {
+            operator,
+            targetValue: value,
+        };
+        this.props.setGlobalFilterValue(id, newFilterValue);
+    }
+
+    onMinimumValueNumericInput(id, value) {
+        const operator = this.filterValue?.operator ?? this.getDefaultOperator();
+        const newFilterValue = {
+            operator,
+            minimumValue: value,
+            maximumValue: this.filterValue?.maximumValue,
```

Comment:
maybe you could re-order the values if the user inputs a min which is greater than the max.
Same below